edit/edit.js

```diff
@@ -508,7 +517,7 @@ window.onbeforeunload = () => {
   if (isCleanGlobal()) {
     return;
   }
-  updateLintReport(null, 0);
+  updateLintReportIfEnabled(null, 0);
   return confirm(t('styleChangesNotSaved'));
```

Comment:
er, disregard the above comment, it's about the old code, we'll handle it in the future